app/src/main/res/values/strings.xml

```diff
@@ -240,4 +240,9 @@ be temporary, and you can try again later.</li>
 
     <string name="error_generic_title">Oops</string>
     <string name="error_generic_message">We can’t load this page for some reason.</string>
+
+    <string name="menu_link_share">Share link</string>
+    <string name="menu_link_copy">Copy link address</string>
+    <string name="menu_image_share">Share image</string>
+    <string name="menu_image_copy">Copy image address</string>
```

Comment:
Maybe make it more obvious that those strings are shown in a context menu (as opposed to the overflow menu) - either by renaming the string or adding a comment.